packages/python/plotly/requires-pandas.txt

```diff
@@ -0,0 +1,2 @@
+pandas; python_version <= '3.8' # Loose requirement for older Python as older numpy is not supported
+pandas>=2.2.0; python_version > '3.0' # Minimum pin to ensure compatibility with all versions of numpy
```

Comment:
```suggestion
pandas>=2.2.0; python_version > '3.8' # Minimum pin to ensure compatibility with all versions of numpy
```
Otherwise this will combine with the other one and this will win for all Py3.x, right?

Reply:
Yep... typo... good catch

Reply:
Actually I realize this is just wrong. Should be <3.9; >=3.9 rather than <=3.9; >3.8. AFK but I'll fix when I'm back.